subsys/net/lib/downloader/src/transports/http.c

```diff
@@ -19,7 +19,11 @@
 
 LOG_MODULE_DECLARE(downloader, CONFIG_DOWNLOADER_LOG_LEVEL);
 
-/* nRF91 modem TLS secure socket buffer limited to 2kB including header */
+/* nRF91 modem TLS secure socket buffer limited to around 2 kB.
+ * If the header and data is too large, the downloader will reduce the range requests until
+ * requirement is satisfied. The application can also set the range_override in
```

Comment:
It's referred to as `requirements` in other documentation sentences of this PR
```suggestion
 * the requirements are satisfied. The application can also set the range_override in
```
@divipillai